extension/lib/shopify/collection-repository/command/GetGraphQL.js

```diff
@@ -0,0 +1,27 @@
+// const query = require('./Get-GraphQL-query')
```

Comment:
You have probably a merge conflict here. This file is deleted in branch WEBC-625. Is there a reason for you to add this file? Maybe try to merge the latest WEBC-625 into your branch. If the file is still in your branch you have had probably a merge conflict